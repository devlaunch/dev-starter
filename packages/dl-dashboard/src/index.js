import React from 'react';
import ReactDOM from 'react-dom';

import DashApp from './app';
import * as serviceWorker from './serviceWorker';
import 'antd/dist/antd.css';

ReactDOM.render(<DashApp />, document.getElementById('root'));

// If you want your app to work offline and load faster, you can change
// unregister() to register() below. Note this comes with some pitfalls.
// Learn more about service workers: http://bit.ly/CRA-PWA
serviceWorker.register();
