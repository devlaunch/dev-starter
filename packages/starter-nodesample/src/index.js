import sum from "./lib/sum";
import dbconnect from "./lib/dbconnect";

async function main() {
  console.log(`Sum Results: ${sum(1, 2)}`);
  try {
    const result = await dbconnect(true);
    console.log(` Promise resolved : ${result} `);
  } catch (err) {
    console.log(`Promise rejected ${err.toString()} `);
  }
}

main();
