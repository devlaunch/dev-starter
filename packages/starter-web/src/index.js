import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';
import { Route, Switch } from 'react-router-dom';
import { ConnectedRouter } from 'react-router-redux';

import App from './containers/App';
import { store, history } from './modules/redux/store';

ReactDOM.render(
  <Provider store={store}>
    <ConnectedRouter history={history}>
      <Switch>
        <Route path="/" component={App} />
      </Switch>
    </ConnectedRouter>
  </Provider>,
  document.getElementById('root'),
);
