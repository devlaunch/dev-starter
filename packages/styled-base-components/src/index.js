import { A } from './components/A';
import { Abbr } from './components/Abbr';
import { Address } from './components/Address';
import { Article } from './components/Article';
import { Aside } from './components/Aside';
import { B } from './components/B';
import { Blockquote } from './components/Blockquote';
import { Button } from './components/Button';
import { Caption } from './components/Caption';
import { Code } from './components/Code';
import { Dd } from './components/Dd';
import { Dfn } from './components/Dfn';
import { Dialog } from './components/Dialog';
import { Dl } from './components/Dl';
import { Dt } from './components/Dt';
import { Fieldset } from './components/Fieldset';
import { Figcaption } from './components/Figcaption';
import { Figure } from './components/Figure';
import { Footer } from './components/Footer';
import { H1 } from './components/H1';
import { H2 } from './components/H2';
import { H3 } from './components/H3';
import { H4 } from './components/H4';
import { H5 } from './components/H5';
import { H6 } from './components/H6';
import { Header } from './components/Header';
import { Hgroup } from './components/Hgroup';
import { Hr } from './components/Hr';
import { Img } from './components/Img';
import { Input } from './components/Input';
import { Kbd } from './components/Kbd';
import { Legend } from './components/Legend';
import { Label } from './components/Label';
import { Main } from './components/Main';
import { Nav } from './components/Nav';
import { Ol } from './components/Ol';
import { Optgroup } from './components/Optgroup';
import { Output } from './components/Output';
import { P } from './components/P';
import { Pre } from './components/Pre';
import { Progress } from './components/Progress';
import { Samp } from './components/Samp';
import { Section } from './components/Section';
import { Select } from './components/Select';
import { Small } from './components/Small';
import { Strong } from './components/Strong';
import { Sub } from './components/Sub';
import { Summary } from './components/Summary';
import { Sup } from './components/Sup';
import { Svg } from './components/Svg';
import { Table } from './components/Table';
import { Textarea } from './components/Textarea';
import { Th } from './components/Th';
import { Ul } from './components/Ul';

export {
  A,
  Abbr,
  Address,
  Article,
  Aside,
  B,
  Blockquote,
  Button,
  Caption,
  Code,
  Dd,
  Dfn,
  Dialog,
  Dl,
  Dt,
  Fieldset,
  Figcaption,
  Figure,
  Footer,
  H1,
  H2,
  H3,
  H4,
  H5,
  H6,
  Header,
  Hgroup,
  Hr,
  Img,
  Input,
  Kbd,
  Legend,
  Label,
  Main,
  Nav,
  Ol,
  Optgroup,
  Output,
  P,
  Pre,
  Progress,
  Samp,
  Section,
  Select,
  Small,
  Strong,
  Sub,
  Summary,
  Sup,
  Svg,
  Table,
  Textarea,
  Th,
  Ul,
};
