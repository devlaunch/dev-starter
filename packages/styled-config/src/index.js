import screenSize from './config/screen-size';

export default screenSize;
