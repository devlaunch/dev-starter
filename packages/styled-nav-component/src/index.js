import { Nav } from './components/Nav';
import { NavLink } from './components/NavLink';

export {
  Nav,
  NavLink,
};
