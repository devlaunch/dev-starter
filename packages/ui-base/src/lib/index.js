import { A } from "./components/Link/A";

import { Abbr } from "./components/Text/Abbr";
import { Address } from "./components/Text/Address";
import { B } from "./components/Text/B";
import { Blockquote } from "./components/Text/Blockquote";
import { Code } from "./components/Text/Code";
import { Display1 } from "./components/Text/Display1";
import { Display2 } from "./components/Text/Display2";
import { Display3 } from "./components/Text/Display3";
import { Display4 } from "./components/Text/Display4";
import { H1 } from "./components/Text/H1";
import { H2 } from "./components/Text/H2";
import { H3 } from "./components/Text/H3";
import { H4 } from "./components/Text/H4";
import { H5 } from "./components/Text/H5";
import { H6 } from "./components/Text/H6";
import { Hgroup } from "./components/Text/Hgroup";
import { Initialism } from "./components/Text/Initialism";
import { Kbd } from "./components/Text/Kbd";
import { Lead } from "./components/Text/Lead";
import { Mark } from "./components/Text/Mark";
import { P } from "./components/Text/P";
import { Pre } from "./components/Text/Pre";
import { Samp } from "./components/Text/Samp";
import { Small } from "./components/Text/Small";
import { Strong } from "./components/Text/Strong";
import { Sub } from "./components/Text/Sub";
import { Sup } from "./components/Text/Sup";

import { Article } from "./components/Articles/Article";
import { Aside } from "./components/Articles/Aside";
import { Main } from "./components/Articles/Main";
import { Nav } from "./components/Articles/Nav";
import { Footer } from "./components/Articles/Footer";
import { Header } from "./components/Articles/Header";
import { Summary } from "./components/Articles/Summary";
import { Section } from "./components/Articles/Section";

import { Button } from "./components/Form/Button";
import { Fieldset } from "./components/Form/Fieldset";

import { Caption } from "./components/Tables/Caption";
import { Table } from "./components/Tables/Table";
import { Textarea } from "./components/Form/Textarea";
import { Th } from "./components/Tables/Th";

import { Dd } from "./components/Lists/Dd";
import { Dfn } from "./components/Lists/Dfn";
import { Dl } from "./components/Lists/Dl";
import { Dt } from "./components/Lists/Dt";
import { Ol } from "./components/Lists/Ol";
import { Ul } from "./components/Lists/Ul";

import { Figcaption } from "./components/Image/Figcaption";
import { Figure } from "./components/Image/Figure";
import { Img } from "./components/Image/Img";
import { Svg } from "./components/Image/Svg";

import { Input } from "./components/Form/Input";
import { Legend } from "./components/Form/Legend";
import { Label } from "./components/Form/Label";
import { Optgroup } from "./components/Form/Optgroup";
import { Output } from "./components/Form/Output";
import { Progress } from "./components/Form/Progress";
import { Select } from "./components/Form/Select";

import { Box } from "./components/Layout/Box";
import { Flex } from "./components/Layout/Flex";

import { BootstrapBaseCss } from "./components/BootstrapBaseCss";
import { Hr } from "./components/Hr";

export {
  A,
  Abbr,
  Address,
  Article,
  Aside,
  B,
  Blockquote,
  Button,
  Caption,
  Code,
  Dd,
  Dfn,
  Display1,
  Display2,
  Display3,
  Display4,
  Dl,
  Dt,
  Fieldset,
  Figcaption,
  Figure,
  Footer,
  BootstrapBaseCss,
  H1,
  H2,
  H3,
  H4,
  H5,
  H6,
  Header,
  Hgroup,
  Hr,
  Img,
  Input,
  Initialism,
  Kbd,
  Lead,
  Legend,
  Label,
  Mark,
  Main,
  Nav,
  Ol,
  Optgroup,
  Output,
  P,
  Pre,
  Progress,
  Samp,
  Section,
  Select,
  Small,
  Strong,
  Sub,
  Summary,
  Sup,
  Svg,
  Table,
  Textarea,
  Th,
  Ul,
  Box,
  Flex
};
