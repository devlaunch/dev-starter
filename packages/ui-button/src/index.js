import { Button } from './components/Button';
import { ButtonGroup } from './components/ButtonGroup';
import { ButtonToolbar } from './components/ButtonToolbar';

export { Button, ButtonGroup, ButtonToolbar };
