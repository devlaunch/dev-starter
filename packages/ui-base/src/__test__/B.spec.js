import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import { B } from "../src";

test("Styles matches B", () => {
  const tree = renderer.create(<B />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule("font-weight", "bolder");
});
