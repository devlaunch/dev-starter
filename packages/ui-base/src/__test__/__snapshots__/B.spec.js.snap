// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles matches B 1`] = `
.c0 {
  font-weight: bolder;
}

<aside
  className="c0"
/>
`;
